### ISSUE TITLE
GEOS-IT gridded fields for pre-2016

### ISSUE THREAD

**Issue by aschuh** [geoschem/geos-chem#2538]
### Your name

Andrew Schuh

### Your affiliation

Colorado State University

### Please provide a clear and concise description of your question or discussion topic.

I'm writing for a broader group of modelers as part of the OCO-2 Flux Inversion MIP exercise. THere is interest in running GEOS-IT for this to produce CO2 flux estimates for 2014-2024.  I noticed that there are only regridded products from 2016 onwards.  Is there a way to add 4x5 and 2x2.5 gridded products for GEOS-IT for earlier years? in particular, 2013-2015?

**Comment by yantosca** [comment]
Thanks @aschuh.  I am not sure how far back in time the GEOS-IT fields go.  Tagging @lizziel @yidant.

**Comment by aschuh** [comment]
There appears to be data from 1998-2001 on WUSTL site, just nothing from 2001-2015.  Hopefully that means we can get 2013-2015.

**Comment by yantosca** [comment]
Thanks @aschuh,  I am not sure in which order they are producing the data.  Some years may have been worked on first.  @yidant would know better.

**Comment by lizziel** [comment]
The plan is to have the entire inventory back to 1998 processed. @yidant is doing that work and can provide a status update. GEOS-IT is still beta in GEOS-Chem is the full inventory is not yet ready.

**Comment by aschuh** [comment]
Two questions.  

1) Is there an ability to prioritize 2013-2015 ? We're getting ready to start a large intercomparison effort where I'm guessing there will be a half dozen GCC contributions.  

2) The other question is, is GEOS-IT ready for game time?  In other words, what does "beta" mean?  Should we be concerned about things like mass inconsistencies or things that can affect long lived passive tracer sims?  The new convection scheme seems to reduce existing uncertainty in transport from my tests, which means there should be push to try to use GEOS-IT if possible.

**Comment by lizziel** [comment]
@aschuh, whenever we roll out a new met source we do a thorough benchmark comparing to whatever met we currently use for benchmarking. In this case it would be MERRA2 and we will we run the benchmark comparison for GEOS-IT as part of 14.6.0 benchmarking. We also need to finish doing the data processing and generation of offline emissions. GEOS-IT also contains a bug (along with GEOS-FP post-2020) that we were previously unaware of, specifically that convective precipitation flux is zero. A fix for that will go into 14.5.1. See https://github.com/geoschem/geos-chem/issues/2469 for more information.

**Comment by yidant** [comment]
Hi @aschuh! For your first question - yes! The processing is ongoing and we can prioritize 2013-2015. It is expected to be ready within this week.

**Comment by aschuh** [comment]
@lizziel , got it.  I've followed the "bug" but it shouldn't have any effect on passive tracer sims I think?  However, reading between the lines a bit, I hear "it probably should be fine and has checked out in benchmarking but is very new and we probably can't TOTALLY vouch for it until it has been used by folks for a bit", is that fair?  My plan would be to use it for the OCO2 inversion work but I'm thinking that some folks, like JPL, might stick to MERRA2.

**Comment by lizziel** [comment]
@aschuh, I've been told the bug affects computation of cloud height which can have impact on mixing. In that sense transport tracers will be affected. I am still working on the project comparing GCHP, GC-Classic, and GEOS transport using GEOS-IT and we plan to redo the offline runs with the fix for this reason.

**Comment by lizziel** [comment]
> However, reading between the lines a bit, I hear "it probably should be fine and has checked out in benchmarking but is very new and we probably can't TOTALLY vouch for it until it has been used by folks for a bit", is that fair?

Yes, I agree with this statement. Note also that I did a comparison already of FP, IT, and MERRA2 transport tracers and IT was very similar to FP. The assumption is because they both use Grell-Freitas convection, unlike MERRA2, and they also both contain the convective mass flux bug (unknown at the time of my comparison). It is yet to be seen how much the bug fix will help bring the results to matching MERRA2, but it is safe to say that IT is comparable to FP starting in June 2020. Whether you use IT would depend on if you are comfortable with the convection switch in GEOS.
